src/Symfony/Bundle/FrameworkBundle/composer.json

```diff
@@ -47,7 +47,7 @@
         "symfony/form": "~2.8|~3.0",
         "symfony/expression-language": "~2.8|~3.0",
         "symfony/process": "~2.8|~3.0",
-        "symfony/serializer": "~2.8|^3.0",
+        "symfony/serializer": "~3.2",
```

Comment:
Are these checks the ones you are talking about? https://github.com/symfony/symfony/blob/35b0ab95270aacdc433dcf9ddbc6bd62021cd3ee/src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php#L993

If these are the only one (I found no other related to the Serializer but I may be wrong), I can create a PR to remove the checks.